docs/Advanced Features/workflows.md

```diff
@@ -0,0 +1,143 @@
+*New in Quilt 3.3*
```

Comment:
@akarve 
could you check this?
Config and schemas are in the bucket, so all these commands work and you can play with it if you want.